Ignoring accents when searching keywords

glenl opened the pull request:
Added a UTF-8 level 1 collator for matching. Since this increases search times I added a search time benchmark output at the bottom of the page.

glenl committed:
Use UTF-8 level 1 collator for matching, display basic search performance.

chrissawer commented:
Would be good to have some benchmarking on this - if it increases CPU usage on the server we may need to check with Eric who owns the server that it is acceptable.

glenl commented:
I absolutely agree. I checked in this branch so that it was available for anybody wanted to review and give feedback.

This branch is a fairly simple attack at the problem. I added some simple benchmarking, added the perl Collation routines, compared times, added simple performance improvements, then stopped unsatisfied.

My conclusion is that it is not wise to back-fill this sort of matching with the existing data cache --- if your original requirements had diacritical-insensitive search at the top of the list you would have developed a more effective data cache design.

glenl commented:
Keeping the branch but closing the pull request.
